### DISCUSSION TITLE
New connectors are not seen in 1.6 UI

### DISCUSSION THREAD

**Discussion by brown3007** [open-metadata/OpenMetadata#19145] [Q&A] [answered]
Hi all,

I'm trying to build new connectors for custom pipeline services.
I use instructions from [here](https://docs.open-metadata.org/latest/developers/contribute/developing-a-new-connector)
When I do it in 1.5, everything is OK, I can create new connections through UI, see their custom logos etc.
But when I try to do in in 1.6, I can see no new connectors in UI menu.
By the way, I can create these new services from API, though they have default logo (gear image).
I tried generating typescript types:
`yarn run json2ts` is not working from #18900
I got paths for new and updated files from `openmetadata-spec/src/main/resources/json/schema/` and ran `sh json2ts.sh <path>` for each file.
Could you help me, what could I miss to do or do wrong?
Thank you!

**Comment by Prajwal214** [comment]
@brown3007 here have been some updates to the documentation. We will make the necessary changes and inform you once they are complete.

**Comment by ayush-shah** [comment]
@brown3007 add those files in staging
and then run:

```
changed_files=$(git diff --cached --name-only --diff-filter=ACM | grep 'openmetadata-spec/src/main/resources/json/schema/')
./openmetadata-ui/src/main/resources/ui/json2ts.sh $changed_files
```

**Reply by brown3007** [reply, marked as answer]
Did not help until I ran:
./openmetadata-ui/src/main/resources/ui/json2ts.sh openmetadata-spec/src/main/resources/json/schema/api/services/createPipelineService.json

Then everything went correct. Thank you!
